packages/devtools_app/lib/src/memory/memory_tracker_model.dart

```diff
@@ -97,7 +97,7 @@ class _TrackerCountColumn extends ColumnData<Tracker> {
   _TrackerCountColumn()
       : super(
           'Count',
-          alignment: ColumnAlignment.right,
+          alignment: ColumnAlignment.left,
```

Comment:
Is this correct?

```
0
10
200
5000
```

not

```
   0
  10
 200
5000
```

Reply:
I changed it, because I thought there is more space on right. Other parameters uses .right, so changing it back to make it more consistent